boards/arm/arty/arty_a7_arm_designstart_m1.dts

```diff
@@ -150,6 +150,48 @@
 			reg = <0x20000000 DT_SIZE_K(32)>;
 		};
 
+		daplink_gpio0: gpio@40010000 {
```

Comment:
Is this something that should have been added when GPIO was added?  It looks like it belongs in a separate commit.

Reply:
@pabigot Thank you for reviewing! The `daplink_gpio0` controller is used for multiplexing access to the off-board QSPI SPI NOR flash (support for this will follow in a separate PR) between read-only XIP access and `jedec,spi-nor` driver access.

I have removed the GPIO node for now and will (re-)introduce it along with support for the MUX.